scout/server/blueprints/variant/utils.py

```diff
@@ -8,12 +8,12 @@
 
 def add_panel_specific_gene_info(panel_info):
     """Adds manualy curated information from a gene panel to a gene
-    
+
```

Comment:
Is this black versions, or editor malfunctions?

Reply:
BLAAAAACK